machine-learning/gunicorn.conf.py

```diff
@@ -0,0 +1,28 @@
+"""
+Gunicorn configuration options.
+https://docs.gunicorn.org/en/stable/settings.html
+"""
+import os
+
+
+# Set the bind address based on the env
+port = os.getenv('MACHINE_LEARNING_PORT') or "3003"
+bind = f"0.0.0.0:{port}"
```

Comment:
Is it possible to configure this so it will also bind on ipv6 if that is available?

Reply:
Ill have a look

Reply:
Thought it was best to just make this configurable in the environment - people can use IPv6 if they need to that way